### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jmsquare**
## Type of Change

- [x] 📚 Examples / docs / tutorials / dependencies update
- [ ] 🔧 Bug fix (non-breaking change which fixes an issue)
- [ ] 🥂 Improvement (non-breaking change which improves an existing feature)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🔐 Security fix

## Checklist

- [x] I've read the [`CODE_OF_CONDUCT.md`](https://github.com/Giskard-AI/ai-inspector/blob/master/CODE_OF_CONDUCT.md) document.
- [x] I've read the [`CONTRIBUTING.md`](https://github.com/Giskard-AI/ai-inspector/blob/master/CONTRIBUTING.md) guide.
- [ ] I've updated the code style using `make codestyle`.
- [ ] I've written tests for all new methods and classes that I created.
- [ ] I've written the docstring in Google format for all the methods and classes that I used.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Additionally, it is helpful if you help us spread the word by mentioning the library in blog articles about the amazing projects it enabled, tweeting about it on occasion, or just staring the repository! ⭐️
+
+If you choose to contribute, please be mindful to respect our [code of conduct](https://github.com/Giskard-AI/giskard/blob/main/CODE_OF_CONDUCT.md).
+
+## The different ways you can contribute to Giskard!
+
+There are 4 ways you can contribute to Giskard:
+* Fixing outstanding issues with the existing code;
+* Implementing new tests;
```
Comment (alexcombessie) [inline CONTRIBUTING.md:13]: I would be more explicit that we mean tests for ML models

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Everyone is encouraged to contribute, and we appreciate each and every one of them. Helping the community is thus not limited to writing code. The community will greatly benefit from your efforts to provide clarification, assist others, and enhance the documentation. 📗
+
+Additionally, it is helpful if you help us spread the word by mentioning the library in blog articles about the amazing projects it enabled, tweeting about it on occasion, or just staring the repository! ⭐️
+
+If you choose to contribute, please be mindful to respect our [code of conduct](https://github.com/Giskard-AI/giskard/blob/main/CODE_OF_CONDUCT.md).
+
+## The different ways you can contribute to Giskard!
+
+There are 4 ways you can contribute to Giskard:
```
Comment (alexcombessie) [inline CONTRIBUTING.md:11]: I would order by importance
1. Submitting issues
2. Contributing to examamples
3. Fixing issues
4. Implementing tests

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+* Fixing outstanding issues with the existing code;
+* Implementing new tests;
+* Contributing to the examples or to the documentation;
+* Submitting issues related to bugs or desired new features.
+
+
+### Did you find a bug? 
+
+First, we would really appreciate it if you could **make sure the bug was not
```
Comment (alexcombessie) [inline CONTRIBUTING.md:20]: That would suppose our bug tracker is public on Github. I really don't think the current tool youtrack is fit for an open-source community.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+already reported** (use the search bar on Github under Issues).
+
+If you did not find it, please follow these steps to inform us:
+
+* Include your **OS type and version**, the versions of **Python**, and different Python libraries you used;
+* A short, self-contained, code snippet that allows us to reproduce the bug in less than 30s;
+* Provide the *full* traceback if an exception is raised.
```
Comment (alexcombessie) [inline CONTRIBUTING.md:27]: I think you mean stacktrace. Not sure what traceback means.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+To get the logs of a given container, you can run
+
+```bash
+docker logs <NAME OF THE CONTAINER>
+```
+
+### Do you want to implement a new test?
```
Comment (alexcombessie) [inline CONTRIBUTING.md:35]: I would say "ML test"

File: CONTRIBUTING.md (hunk 1 above)
Comment (alexcombessie) [reply CONTRIBUTING.md:13]: And also add a separate point for adding features

**Review by alexcombessie — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 4 above)
Comment (jmsquare) [reply CONTRIBUTING.md:27]: Both words are synonyms (https://realpython.com/python-traceback/) and are used the same way.
But ok, let's use stack trace

File: CONTRIBUTING.md (hunk 3 above)
Comment (jmsquare) [reply CONTRIBUTING.md:20]: ok let's discuss it together about whether YouTrack is adapted to open source or not

**Review by alexcombessie — APPROVED** [review]

**Comment by alexcombessie** [comment]
Awesome, thanks @jmsquare ! I let you merge 👍 

For the topic on what issue tracker we should use to facilitate contribution, do you want to start a Github discussion?

**Pull request merged**